lib/src/model/tile_model.dart

```diff
@@ -1,5 +1,8 @@
+import 'dart:collection';
 import 'dart:ui';
 
+import 'package:vector_tile_renderer/src/model/path_utils.dart';
```

Comment:
we're going with relative imports in this project.

Reply:
done.